.github/workflows/_integration-tests.yml

```diff
@@ -378,7 +378,8 @@ jobs:
         -   uses: actions/setup-python@v5
             with:
                 python-version: ${{ inputs.python-version }}
-        -   run: echo "HATCH_PYTHON=${{ inputs.python-version }}" >> $GITHUB_ENV
+        # TODO: update 3.10 to ${{ inputs.python-version }} once we update default python to >=3.10
+        # the version of dagger-io that we use does not support py3.9
+        -   run: echo "HATCH_PYTHON=3.10" >> $GITHUB_ENV
         -   uses: pypa/hatch@install
-        -   run: hatch run pip install -r dagger/requirements.txt
```

Comment:
This was moved into the integration test call to make sure it's installed in the correct environment. That could also be done here, but it felt more appropriate to be in the script instead of here. Also, the default environment installs it via `hatch run setup`, so now it's all in the same file.